x-pack/plugins/actions/server/create_unsecured_execute_function.test.ts

```diff
@@ -21,58 +21,63 @@ const internalSavedObjectsRepository = savedObjectsRepositoryMock.create();
 beforeEach(() => jest.resetAllMocks());
 
 describe('bulkExecute()', () => {
-  test('schedules the actions with all given parameters with a preconfigured connector', async () => {
-    const executeFn = createBulkUnsecuredExecutionEnqueuerFunction({
-      taskManager: mockTaskManager,
-      connectorTypeRegistry: actionTypeRegistryMock.create(),
-      preconfiguredConnectors: [
-        {
-          id: '123',
-          actionTypeId: '.email',
-          config: {},
-          isPreconfigured: true,
-          isDeprecated: false,
-          isSystemAction: false,
-          name: 'x',
-          secrets: {},
-        },
-      ],
-    });
+  test.each([
```

Comment:
All tests in the file as the same as before. The only difference is that they run for twice, one for preconfigured actions and one for system actions.